pycronofy/client.py

```diff
@@ -897,3 +899,20 @@ def delete_availability_rule(self, availability_rule_id):
         :param string availability_rule_id: ID of the Availability Rule to delete.
         """
         self.request_handler.delete(endpoint='availability_rules/%s' % availability_rule_id)
+
+    def hmac_match(self, hmac_string, body):
+        if hmac_string is None or not hmac_string:
```

Comment:
Please write some doc comments similar to the other methods

Reply:
Suggestion applied! :)